frontend/frontend/honeylogger/__init__.py

```diff
@@ -78,6 +78,7 @@ def end(self) -> None:
 
 class SSHSession(Session, Protocol):
     """Representation of an attacker's SSH session while being connected to the honeypot."""
+    session_id: Optional[int]
```

Comment:
It would be nice if this was not exposed as a part of the public interface for a logging session since the fact that there is an id, especially with this specific type, is really an implementationen detail. I noticed it's needed in ProxyHandler, however. I'm not sure how to deal with that, so if it's too much hassle, it's not a big deal.

Reply:
The only thing I can think of is overriding `__str__` to something like "Session: x" where x would be the id, which I don't think will have any consequences. What do you think about doing it that way?

Reply:
I think that makes sense and would be a good solution actually!